package.json

```diff
@@ -4,7 +4,7 @@
   "version": "0.1.0",
   "description": "terra-graphs",
   "engines": {
-    "node": ">=10 <12"
+    "node": ">=12"
```

Comment:
This would mean it also supports Node 14 and 16, which I don't think we want to yet. Dropping Node 10 support would be non-passive. 

Should we change this to `"^10.0.0 || ^12.0.0"` to support only 10 and 12?